src/Storage/Helpers/InstanceHelper.cs

```diff
@@ -267,8 +264,8 @@ public static (string InstanceOwnerIdType, string InstanceOwnerIdValue) GetIdent
                 return (string.Empty, string.Empty);
             }
 
-            partyType = parts[0];
-            partyNumber = parts[1];
+            string partyType = parts[0];
+            string partyNumber = parts[1];
 
             string[] partyTypeHayStack = ["person", "organization"];
```

Comment:
I don't see how organization with a Z would work here. Missing tests that would have caught the issue as well. A test that starts at the controller level with org and/or person as input.

Reply:
Still missing the test on controller level.

Reply:
Well, I don't think we need a test at the controller level as this method follows the principle of pure function(input/output). However, we definitely can introduce a test with the `z` spelling of organisation to the UnitTest level which will cover the scenario! Beside its way simpler than a test at the controller level. 🙂 

```
[InlineData("organisation:12345", "organisation", "12345")]
[InlineData("organization:12345", "", "")]
```

Reply:
You currently dont have any tests actually validating that the code you've written is called from anywhere. What happens if someone deletes the code lines from the controller when re-factoring?

Reply:
Yea! Valid point! 🤔 👍